### PR TITLE
Fix localisation in System Configuration View

### PR DESCRIPTION
**Pull request by laverman**

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java [hunk 1]
```diff
@@ -27,7 +28,7 @@
     private static final String DISTRIBUTION_BAR_PART_MAIN_STYLE = GwtDistributionBar.CLASSNAME + "-part";
     private static final String DISTRIBUTION_BAR_PART_CLASSNAME_PREFIX = GwtDistributionBar.CLASSNAME + "-part-";
     private static final String DISTRIBUTION_BAR_PART_VALUE_CLASSNAME = GwtDistributionBar.CLASSNAME + "-value";
-    private static final String UNINITIALIZED_VALUE_CLASSNAME = GwtDistributionBar.CLASSNAME + "-uninitizalized";
+    private static final String UNINITIALIZED_VALUE_CLASSNAME = GwtDistributionBar.CLASSNAME + "-uninitialized";
```
Comment (laverman) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java]: Verify if this is used somewhere else

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java [hunk 2]
```diff
@@ -69,27 +73,57 @@ public static String getDistributionBarAsHTMLString(final Map<Status, Long> stat
...
         return tooltip.toString();
     }
 
+    private static String getLabel(final Status status, final VaadinMessageSource i18n) {
+
+        String label;
+
+        switch (status) {
+
+            case SCHEDULED: label = "label.scheduled";
```
Comment (laverman) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java]: check formatting

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java [hunk 3]
```diff
@@ -69,27 +73,57 @@ public static String getDistributionBarAsHTMLString(final Map<Status, Long> stat
...
+                break;
+            case ERROR: label = "label.error";
+                break;
+            case FINISHED: label = "label.finished";
+                break;
+            case CANCELLED: label = "label.cancelled";
+                break;
+            case NOTSTARTED: label = "label.notStarted";
+                break;
+            default:
```
Comment (laverman) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/DistributionBarHelper.java]: Better to throw exception e.g. illegalStateExc.

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/authentication/GatewaySecurityTokenAuthenticationConfigurationItem.java [hunk 4]
```diff
@@ -58,7 +58,7 @@ public GatewaySecurityTokenAuthenticationConfigurationItem(
         detailLayout = new VerticalLayout();
         detailLayout.setImmediate(true);
 
-        final Button gatewaytokenBtn = SPUIComponentProvider.getButton("TODO-ID", "Regenerate Key", "",
+        final Button gatewaytokenBtn = SPUIComponentProvider.getButton("TODO-ID", i18n.getMessage("configuration.button.regenerateKey"), "",
```
Comment (StefanKlt) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/authentication/GatewaySecurityTokenAuthenticationConfigurationItem.java]: Check if "TODO-ID" can be removed. Setting it to null should result in a button without an ID

**Pull request merged**